Source/WebCore/platform/ios/wak/WKUtilities.h

```diff
@@ -59,8 +59,10 @@ struct _WAKObject
 };
 
 const void *WKCreateObjectWithSize (size_t size, WKClassInfo *info);
-const void *WKRetain(const void *object);
-void WKRelease(const void *object);
+// These functions break naming convention to avoid overshadowing WKRetain and WKRelease in WebKit
+// during InstallAPI, in build configurations where WebKit reexports WebCore.
```

Comment:
May be worth putting actual details in the bug, and making the comment smaller by referencing that.

Currently, it's both very long and very vague, probably not much more helpful than "this is for reasons".